_raml/apps/carts/examples/get_carts_cartid_response.json

```diff
@@ -174,8 +174,18 @@
   "paymentData": {
     "paymentMethod": {
       "id": "5171DD89-6DE5-E758-E046-AC14145F9009",
-      "name": "PayPal",
-      "providerName": "PayPal"
+      "name": "PayPal, Direct debit, Credit card",
+      "providerName": "PayPal PLUS",
+      "additionalData": {
```

Comment:
I managed to get real data:
"InvoiceBankName": "Deutsche Bank"
"InvoiceAccountHolderName": "PayPal Europe"
"InvoiceIBAN", "DE98120700888000554718", (maybe this one should be fake as you already did, but with this format)
"InvoiceReferenceNumber": "87H55148KK9907810"
"InvoicePaymentDueDate": "2020-03-27T08:16:30.000Z"
"InvoiceBIC": "DEUTDEDBPAL"
"", "",
"", ""

Reply:
Thank you for the input @manel-ePages :) So invoiceAmount and invoiceCurrency are not provided?

Reply:
Yes they are provided (as you already had). It's just that didn't include because they can stay as they are now.
(note: InvoiceBankName, InvoiceAccountHolderName and InvoiceBIC are related apparently, so better change by the ones I provide).